src/algorithms/time_evolution/evoltools.jl

```diff
@@ -181,27 +181,28 @@ Obtain the 3-site gate MPO on the southeast cluster at position `[row, col]`
         c      c+1
 ```
 """
-function _get_gatempo_se(gate::LocalOperator, row::Int, col::Int)
-    Nr, Nc = size(gate.lattice)
+function _get_gatempo_se(ham::LocalOperator, dt::Number, row::Int, col::Int)
+    Nr, Nc = size(ham.lattice)
     @assert 1 <= row <= Nr && 1 <= col <= Nc
-    unit = id(space(gate.terms[1].second, 1))
+    unit = id(space(ham.terms[1].second, 1))
     sites = (
         CartesianIndex(row, col),
         CartesianIndex(row, col + 1),
         CartesianIndex(row - 1, col + 1),
     )
-    nb1x = get_gateterm(gate, (sites[1], sites[2]))
-    nb1y = get_gateterm(gate, (sites[2], sites[3]))
-    nb2 = get_gateterm(gate, (sites[1], sites[3]))
+    nb1x = get_gateterm(ham, (sites[1], sites[2]))
+    nb1y = get_gateterm(ham, (sites[2], sites[3]))
+    nb2 = get_gateterm(ham, (sites[1], sites[3]))
     op = (1 / 2) * (nb1x ⊗ unit + unit ⊗ nb1y) + permute(nb2 ⊗ unit, ((1, 3, 2), (4, 6, 5)))
+    op = exp(-dt * op)
```

Comment:
```suggestion
    @tensor Odt[i' j' k'; i j k] := -dt * ((nb1x[i' j'; i j] * unit[k' k] + unit[i'; i] * nb1y[j' k'; j k]) / 2 + nb2[i' k'; i k] * unit[j'; j])
    op = exp(Odt)
```
This is just a suggestion because I think it reads slightly easier, but also because it avoids a bunch of intermediate objects

Reply:
I'll apply this in next push.